src/test/java/com/epam/jamp2/service/impl/CalculationCommandExecutionServiceImplTest.java

```diff
@@ -0,0 +1,95 @@
+package com.epam.jamp2.service.impl;
+
+import com.epam.jamp2.model.CalculationCommand;
+import com.epam.jamp2.model.CommandFormatException;
+import com.epam.jamp2.model.UnknownCurrencyException;
+import com.epam.jamp2.model.Value;
+import com.epam.jamp2.service.CalculationCommandExecutionService;
+import com.epam.jamp2.service.FxRatesService;
+import org.junit.AfterClass;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.runners.MockitoJUnitRunner;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.Optional;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.mockito.Matchers.any;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+
+@RunWith(MockitoJUnitRunner.class)
+public class CalculationCommandExecutionServiceImplTest {
+  private static CalculationCommandExecutionService calculationService;
+
+  @Mock
+  FxRatesService mockedFXRatesService;
+
+  @InjectMocks
+  private CalculationCommandExecutionService mockedCalculationService = new CalculationCommandExecutionServiceImpl();
```

Comment:
1. Why do you need two instances of CalculationCommandExecutionService in this Test class?
2. Using "mocked" prefix in "mockedCalculationService" is incorrect since you don't mock the object (what you did is injected mocked dependencies into it) - it is confusing. Also I would recommend to rename it to "underTest" which is a common practice I saw in my previous projects.